paasta_tools/setup_istio_mesh.py

```diff
@@ -172,6 +172,37 @@ def setup_paasta_namespace_services(
     return status
 
 
+def cleanup_kube_svc(
+    kube_client: KubeClient,
+    smartstack_namespaces: AbstractSet,
```

Comment:
why isn't this just `Set[str]`?

Reply:
https://github.com/Yelp/paasta/blob/340344c3765df36de69468b8bfc4a5d273b2c3bb/paasta_tools/setup_istio_mesh.py#L225 .keys() returns a `AbstractSet` object, which cause mypy tests to fail in case of `Set[str]`

Reply:
right - if you walk back to `load_smartstack_namespaces` you see we just return a `Mapping` rather than a more constrained type.

Reply:
yeah, `load_smartstack_namespaces` returns a map of type 


```
{
  service_name: 
    {
      'port':  'port-number'
    }
}

```
To get the list of service names, we use `.keys()` method on the return value of `load_smartstack_namespaces` , which in turn returns an `AbstractSet object`